### ISSUE TITLE
[Zone] Stormpeaks - review of Storyline order - correctness of the questlines causing blocks & RP oddities

### ISSUE THREAD

**Issue by Echinaceax** [WoWManiaUK/Redemption#5736]
[Reference questline documentation as scratchpad for corrections - issues to be documented in comments]

**Links:**
https://wowwiki-archive.fandom.com/wiki/Storm_Peaks_quests

K3
Breadcrumbs coming in:

**Rin Duoctane has from Dalaran:**

N [77] Luxurious Getaway! https://www.wow-mania.com/armory/?quest=12853

**Jeer Sparksocket has:**

N [78] Clean Up https://www.wow-mania.com/armory/?quest=12818
N [78] Just Around the Corner https://www.wow-mania.com/armory/?quest=12819
N [78] Slightly Unstable https://www.wow-mania.com/armory/?quest=12826
N [78] A Delicate Touch https://www.wow-mania.com/armory/?quest=12820

**Ricket has:**

N [78] Expression of Gratitude https://www.wow-mania.com/armory/?quest=12836

Side quest: N [78] Reclaimed Rations https://www.wow-mania.com/armory/?quest=12827

N [78] Ample Inspiration https://www.wow-mania.com/armory/?quest=12828
N [78] Opening the Backdoor https://www.wow-mania.com/armory/?quest=12821
N [78] Know No Fear https://www.wow-mania.com/armory/?quest=12822
N [78] A Flawless Plan https://www.wow-mania.com/armory/?quest=12823
N [78] Demolitionist Extraordinaire https://www.wow-mania.com/armory/?quest=12824
A [78] When All Else Fails https://www.wow-mania.com/armory/?quest=12862 or H [78] When All Else Fails https://www.wow-mania.com/armory/?quest=13060
N [78 Daily] Overstock https://www.wow-mania.com/armory/?quest=12833

**N [78] A Flawless Plan
N [78] Demolitionist Extraordinaire
("When All Else Fails" leads to Frosthold)**

**Crystalweb Cavern**

**Tore Rumblewrench has:**

N [78] Ore Repossession https://www.wow-mania.com/armory/?quest=12830
N [78] Moving In https://www.wow-mania.com/armory/?quest=12829

**Injured Goblin Miner has:**

N [78] Only Partly Forgotten https://www.wow-mania.com/armory/?quest=12831
N [78] Bitter Departure https://www.wow-mania.com/armory/?quest=12832

**Gretchen Fizzlespark has:**

N [79] They Took Our Men! https://www.wow-mania.com/armory/?quest=12843

Side quest: N [79] Equipment Recovery https://www.wow-mania.com/armory/?quest=12844

N [79] Leave No Goblin Behind https://www.wow-mania.com/armory/?quest=12846
N [79] The Crone's Bargain https://www.wow-mania.com/armory/?quest=12841
N [79] Mildred the Cruel https://www.wow-mania.com/armory/?quest=12905
N [79] Discipline https://www.wow-mania.com/armory/?quest=12906
N [79] Examples to be Made https://www.wow-mania.com/armory/?quest=12907
N [79] A Certain Prisoner https://www.wow-mania.com/armory/?quest=12908
N [79] A Change of Scenery https://www.wow-mania.com/armory/?quest=12921
N [80] Is That Your Goblin? https://www.wow-mania.com/armory/?quest=12969
N [80] The Hyldsmeet https://www.wow-mania.com/armory/?quest=12970
N [80] Taking on All Challengers https://www.wow-mania.com/armory/?quest=12971

Prerequisite for daily: N [80 Daily] Defending Your Title https://www.wow-mania.com/armory/?quest=13423

N [80] You'll Need a Bear https://www.wow-mania.com/armory/?quest=12972
N [80] Going Bearback https://www.wow-mania.com/armory/?quest=12851 
N [80] Cold Hearted https://www.wow-mania.com/armory/?quest=12856
N [80] Deemed Worthy https://www.wow-mania.com/armory/?quest=13063
N [80] Making a Harness https://www.wow-mania.com/armory/?quest=12900
N [80] The Last of Her Kind https://www.wow-mania.com/armory/?quest=12983

Side quest: N [80] The Slithering Darkness https://www.wow-mania.com/armory/?quest=12989

N [80] The Warm-Up https://www.wow-mania.com/armory/?quest=12996
N [80] Into the Pit https://www.wow-mania.com/armory/?quest=12997

Prerequisite for daily: N [80 Daily] Back to the Pit https://www.wow-mania.com/armory/?quest=13424

N [80] Prepare for Glory https://www.wow-mania.com/armory/?quest=13061
N [80] Lok'lira's Parting Gift https://www.wow-mania.com/armory/?quest=13062
N [80] The Drakkensryd https://www.wow-mania.com/armory/?quest=12886

Prerequisite for daily: N [80 Daily] Maintaining Discipline https://www.wow-mania.com/armory/?quest=13422

N [80] Sibling Rivalry https://www.wow-mania.com/armory/?quest=13064
N [80] Mending Fences (+28,600 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=12915

Note: The chain continues from a dropped item: Slag Covered Metal https://www.wow-mania.com/armory/?item=41556. You should complete this quest before returning to Thorim.

N [80] The Refiner's Fire https://www.wow-mania.com/armory/?quest=12922
N [80] A Spark of Hope (+28,600 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=12956
N [80] Forging an Alliance (+325 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=12924

**Mandatory side quest:**

N [80] You Can't Miss Him (+97 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=12966
N [80] Battling the Elements (+98 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=12967

Prerequisite for daily: N [80 Daily] Hot and Cold (+455 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=12981

N [80] A New Beginning https://www.wow-mania.com/armory/?quest=13009
N [80] Veranus https://www.wow-mania.com/armory/?quest=13050
N [80] Territorial Trespass https://www.wow-mania.com/armory/?quest=13051
N [80] Krolmir, Hammer of Storms (+455 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=13010

**Mandatory side quest:**

N [80] Jormuttar is Soo Fat... (+325 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=13011
N [80] In Memoriam (+325 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=12975
N [80] A Monument to the Fallen (+97 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=12976

Prerequisite for daily: N [80 Daily] Blowing Hodir's Horn (+455 The Sons of Hodir) https://www.wow-mania.com/armory/?quest=12977

N [80] The Terrace of the Makers https://www.wow-mania.com/armory/?quest=13057
N [80] The Earthen Oath https://www.wow-mania.com/armory/?quest=13005 & N [80] Loken's Lackeys https://www.wow-mania.com/armory/?quest=13035
N [80] The Reckoning https://www.wow-mania.com/armory/?quest=13047
N [80D] Whatever it Takes! https://www.wow-mania.com/armory/?quest=13108 (+650 The Sons of Hodir) & N [80D] Diametrically Opposed https://www.wow-mania.com/armory/?quest=13109

**Brunnhildar Village**

N [80] Aberrations https://www.wow-mania.com/armory/?quest=12925
N [80] Off With Their Black Wings https://www.wow-mania.com/armory/?quest=12942
N [80] Valkyrion Must Burn https://www.wow-mania.com/armory/?quest=12953
N [80] Yulda's Folly https://www.wow-mania.com/armory/?quest=12968
N [79 Daily] Maintaining Discipline https://www.wow-mania.com/armory/?quest=13422 (The Drakkensryd required)

**Dun Niffelem**
Neutral

N [80 Daily] Hot and Cold https://www.wow-mania.com/armory/?quest=12981 (requires Battling the Elements)
N [80 Daily] Blowing Hodir's Horn https://www.wow-mania.com/armory/?quest=12977

Friendly

N [80] Jormuttar is Soo Fat...https://www.wow-mania.com/armory/?quest=13011
N [80] Forging a Head https://www.wow-mania.com/armory/?quest=12985
N [80] Mounting Hodir's Helm https://www.wow-mania.com/armory/?quest=12987
N [80 Daily] Polishing the Helm https://www.wow-mania.com/armory/?quest=13006
N [80] Everfrost https://www.wow-mania.com/armory/?quest=13420
N [80] Remember Everfrost! (repeatable) https://www.wow-mania.com/armory/?quest=13421
N [80] Hodir's Tribute (repeatable) https://www.wow-mania.com/armory/?quest=13559

Honored

N [80 Daily] Spy Hunter https://www.wow-mania.com/armory/?quest=12994
N [80] Raising Hodir's Spear https://www.wow-mania.com/armory/?quest=13001
N [80 Daily] Thrusting Hodir's Spear https://www.wow-mania.com/armory/?quest=13003

Revered

N [80 Daily] Feeding Arngrim https://www.wow-mania.com/armory/?quest=13046

Frosthold
**Rork Sharpchin has:**

A [78] Ancient Relics (repeatable) https://www.wow-mania.com/armory/?quest=12870

Lagnus has:

A [78] Offering Thanks https://www.wow-mania.com/armory/?quest=12863
A [78] Missing Scouts https://www.wow-mania.com/armory/?quest=12864
A [78] Stemming the Aggressors https://www.wow-mania.com/armory/?quest=12866
A [78G3] Sirana Iceshriek https://www.wow-mania.com/armory/?quest=12868

**Fjorlin Frostbrow has:**

A [78] Loyal Companions https://www.wow-mania.com/armory/?quest=12865
A [78] Baby Stealers https://www.wow-mania.com/armory/?quest=12867

**Archaeologist Andorin has:**

A [80] On Brann's Trail https://www.wow-mania.com/armory/?quest=12854
A [80] Sniffing Out the Perpetrator https://www.wow-mania.com/armory/?quest=12855
A [80] Pieces to the Puzzle https://www.wow-mania.com/armory/?quest=12858
A [80] Data Mining https://www.wow-mania.com/armory/?quest=12860
A [80] The Library Console https://www.wow-mania.com/armory/?quest=13415
A [80] Norgannon's Shell https://www.wow-mania.com/armory/?quest=12872
A [80] Aid from the Explorers' League https://www.wow-mania.com/armory/?quest=12871
A [80] The Frostborn King https://www.wow-mania.com/armory/?quest=12873
A [80] Fervor of the Frostborn https://www.wow-mania.com/armory/?quest=12874
A [80] An Experienced Guide https://www.wow-mania.com/armory/?quest=12875

Side quest: A [80] Unwelcome Guests https://www.wow-mania.com/armory/?quest=12876

A [80] The Lonesome Watcher https://www.wow-mania.com/armory/?quest=12877
A [80] Fate of the Titans https://www.wow-mania.com/armory/?quest=12986
A [80] The Hidden Relic https://www.wow-mania.com/armory/?quest=12878
A [80] Fury of the Frostborn King https://www.wow-mania.com/armory/?quest=12879
A [80] The Master Explorer https://www.wow-mania.com/armory/?quest=12880
A [80] The Brothers Bronzebeard https://www.wow-mania.com/armory/?quest=12973
A [78 Daily] Pushed Too Far https://www.wow-mania.com/armory/?quest=12869

**Grom'arsh Crash-site**

H [78] Ancient Relics (repeatable) https://www.wow-mania.com/armory/?quest=12870
H [80] Emergency Measures https://www.wow-mania.com/armory/?quest=13000
H [80] The Missing Tracker https://www.wow-mania.com/armory/?quest=13054
H [80] Cave Medicine https://www.wow-mania.com/armory/?quest=13055
H [80] There's Always Time for Revenge https://www.wow-mania.com/armory/?quest=13056

**Boktar Bloodfury has:**

H [80] The Missing Bronzebeard https://www.wow-mania.com/armory/?quest=12895
H [80] The Nose Knows https://www.wow-mania.com/armory/?quest=12909 
H [80] Sniffing Out the Perpetrator https://www.wow-mania.com/armory/?quest=12910
H [80] Speak Orcish, Man! https://www.wow-mania.com/armory/?quest=12913
H [80] Speaking with the Wind's Voice https://www.wow-mania.com/armory/?quest=12917
H [80] Catching up with Brann https://www.wow-mania.com/armory/?quest=12920
H [80] Pieces of the Puzzle https://www.wow-mania.com/armory/?quest=12926
H [80] Data Mining https://www.wow-mania.com/armory/?quest=12927
H [80] The Library Console https://www.wow-mania.com/armory/?quest=13416
H [80] Norgannon's Shell https://www.wow-mania.com/armory/?quest=12928 
H [80] Going After the Core https://www.wow-mania.com/armory/?quest=13273
H [80] The Core's Keeper https://www.wow-mania.com/armory/?quest=13274
H [80] Forging the Keystone https://www.wow-mania.com/armory/?quest=13285

**Camp Tunka'lo**

H [80] Xarantaur, the Witness https://www.wow-mania.com/armory/?quest=13426
H [80] The Witness and the Hero https://www.wow-mania.com/armory/?quest=13034
H [80] Memories of Stormhoof https://www.wow-mania.com/armory/?quest=13037
H [80] Distortions in Time https://www.wow-mania.com/armory/?quest=13038
H [80] Where Time Went Wrong 
H [80] The Hero's Arms https://www.wow-mania.com/armory/?quest=13049
H [80] Changing the Wind's Course https://www.wow-mania.com/armory/?quest=13058

**Bouldercrag Refuge**
After completing Norgannon's Shell:

H [80] The Earthen of Ulduar https://www.wow-mania.com/armory/?quest=12929 or A [80] The Exiles of Ulduar https://www.wow-mania.com/armory/?quest=12885
N [80] Rare Earth https://www.wow-mania.com/armory/?quest=12930
N [80] Relief for the Fallen https://www.wow-mania.com/armory/?quest=12937 closes N [80] Fighting Back https://www.wow-mania.com/armory/?quest=12931
N [80] Slaves of the Stormforged https://www.wow-mania.com/armory/?quest=12957 closes N [80] The Dark Ore https://www.wow-mania.com/armory/?quest=12964
N [80] The Gifts of Loken https://www.wow-mania.com/armory/?quest=12965 closes N [80] Facing the Storm https://www.wow-mania.com/armory/?quest=12978

While killing golems, loot the [Dark Armor Plate] to continue the quest chain. https://www.wow-mania.com/armory/?item=42203

N [80] Armor of Darkness https://www.wow-mania.com/armory/?quest=12979
N [80] The Armor's Secrets https://www.wow-mania.com/armory/?quest=12980
N [80] Valduran the Stormborn https://www.wow-mania.com/armory/?quest=12984
N [80] Destroy the Forges! https://www.wow-mania.com/armory/?quest=12988 closes N [80] Hit Them Where it Hurts https://www.wow-mania.com/armory/?quest=12991
N [80] A Colossal Threat https://www.wow-mania.com/armory/?quest=12993
N [80] The Heart of the Storm https://www.wow-mania.com/armory/?quest=12998
N [80] The Iron Colossus https://www.wow-mania.com/armory/?quest=13007

**Comment by WhitephoenixTheBunny** [comment]
Okay, Kinda of "ignore" this, as I am doing it on live as ally but this will be a check point


Gretchen Fizzlespark has:

- [ ] N [79] [They Took Our Men! ](https://www.wow-mania.com/armory/?quest=12843)

- [ ] Side quest: N [79] [Equipment Recovery ](https://www.wow-mania.com/armory/?quest=12844)

- [ ] N [79] [Leave No Goblin Behind](https://www.wow-mania.com/armory/?quest=12846) 
- [ ] N [79] [The Crone's Bargain]( https://www.wow-mania.com/armory/?quest=12841)
- [ ] N [79] [Mildred the Cruel ](https://www.wow-mania.com/armory/?quest=12905)
- [ ] N [79] [Discipline](https://www.wow-mania.com/armory/?quest=12906) 
- [ ] N [79] [Examples to be Made ](https://www.wow-mania.com/armory/?quest=12907)
- [ ] N [79] [A Certain Prisoner ](https://www.wow-mania.com/armory/?quest=12908)
- [ ] N [79][ A Change of Scenery ](https://www.wow-mania.com/armory/?quest=12921)
- [ ] N [80] [Is That Your Goblin?]( https://www.wow-mania.com/armory/?quest=12969)
- [ ] N [80] [The Hyldsmeet]( https://www.wow-mania.com/armory/?quest=12970)
- [ ] N [80] [Taking on All Challengers ](https://www.wow-mania.com/armory/?quest=12971)

Prerequisite for daily: - [ ] N [80 Daily] [Defending Your Title]( https://www.wow-mania.com/armory/?quest=13423)

- [ ] N [80] [You'll Need a Bear ](https://www.wow-mania.com/armory/?quest=12972)
- [ ] N [80] [Going Bearback ](https://www.wow-mania.com/armory/?quest=12851)
- [ ] N [80] [Cold Hearted ](https://www.wow-mania.com/armory/?quest=12856)
- [ ] N [80] [Deemed Worthy ](https://www.wow-mania.com/armory/?quest=13063)
- [ ] N [80] [Making a Harness ](https://www.wow-mania.com/armory/?quest=12900)
- [ ] N [80] T[he Last of Her Kind]( https://www.wow-mania.com/armory/?quest=12983)

Side quest: - [ ] N [80] [The Slithering Darkness ](https://www.wow-mania.com/armory/?quest=12989)

- [ ] N [80] [The Warm-Up ](https://www.wow-mania.com/armory/?quest=12996)
- [ ] N [80][ Into the Pit]( https://www.wow-mania.com/armory/?quest=12997)

Prerequisite for daily: - [ ] N [80 Daily][ Back to the Pit ](https://www.wow-mania.com/armory/?quest=13424)

- [ ] N [80] [Prepare for Glory ](https://www.wow-mania.com/armory/?quest=13061)
- [ ] N [80] [Lok'lira's Parting Gift ](https://www.wow-mania.com/armory/?quest=13062)
- [ ] N [80] [The Drakkensryd ](https://www.wow-mania.com/armory/?quest=12886)

Prerequisite for daily: - [ ] N [80 Daily] [Maintaining Discipline ](https://www.wow-mania.com/armory/?quest=13422)

- [ ] N [80] [Sibling Rivalry ](https://www.wow-mania.com/armory/?quest=13064)
- [ ] N [80] [Mending Fences](https://www.wow-mania.com/armory/?quest=12915) (+28,600 The Sons of Hodir) 

Note: The chain continues from a dropped item:[ Slag Covered Metal .](https://www.wow-mania.com/armory/?item=41556) You should complete this quest before returning to Thorim.

- [ ] N [80] [The Refiner's Fire]( https://www.wow-mania.com/armory/?quest=12922)
- [ ] N [80] [A Spark of Hope](https://www.wow-mania.com/armory/?quest=12956) (+28,600 The Sons of Hodir) 
- [ ] N [80] [Forging an Alliance ](https://www.wow-mania.com/armory/?quest=12924)(+325 The Sons of Hodir) 

Mandatory side quest:

- [ ] N [80] [You Can't Miss Him](https://www.wow-mania.com/armory/?quest=12966) (+97 The Sons of Hodir) 
- [ ] N [80] [Battling the Elements](https://www.wow-mania.com/armory/?quest=12967) (+98 The Sons of Hodir) 

Prerequisite for daily: - [ ] N [80 Daily] [Hot and Cold ](https://www.wow-mania.com/armory/?quest=12981)(+455 The Sons of Hodir) 

- [ ] N [80] [A New Beginning ](https://www.wow-mania.com/armory/?quest=13009)
- [ ] N [80][ Veranus ](https://www.wow-mania.com/armory/?quest=13050)
- [ ] N [80] [Territorial Trespass]( https://www.wow-mania.com/armory/?quest=13051)
- [ ] N [80] [Krolmir, Hammer of Storms ](https://www.wow-mania.com/armory/?quest=13010)(+455 The Sons of Hodir) 

Mandatory side quest:

- [ ] N [80] [Jormuttar is Soo Fat... ](https://www.wow-mania.com/armory/?quest=13011)(+325 The Sons of Hodir) 
- [ ] N [80][ In Memoriam]( https://www.wow-mania.com/armory/?quest=12975) (+325 The Sons of Hodir)
- [ ] N [80] [A Monument to the Fallen]( https://www.wow-mania.com/armory/?quest=12976) (+97 The Sons of Hodir)

Prerequisite for daily: - [ ] N [80 Daily][ Blowing Hodir's Horn ](https://www.wow-mania.com/armory/?quest=12977)(+455 The Sons of Hodir) 

- [ ] N [80] [The Terrace of the Makers ](https://www.wow-mania.com/armory/?quest=13057)
- [ ] N [80] [The Earthen Oath &](https://www.wow-mania.com/armory/?quest=13005 ) N closes [80][ Loken's Lackeys ](https://www.wow-mania.com/armory/?quest=13035)
- [ ] N [80] [The Reckoning]( https://www.wow-mania.com/armory/?quest=13047)
- [ ] N [80D] [Whatever it Takes! ](https://www.wow-mania.com/armory/?quest=13108) (+650 The Sons of Hodir) & - [ ] N [80D] [Diametrically Opposed ](https://www.wow-mania.com/armory/?quest=13109)

Brunnhildar Village

- [ ] N [80] [Aberrations ](https://www.wow-mania.com/armory/?quest=12925)
- [ ] N [80] [Off With Their Black Wings]( https://www.wow-mania.com/armory/?quest=12942)
- [ ] N [80][ Valkyrion Must Burn]( https://www.wow-mania.com/armory/?quest=12953)
- [ ] N [80] [Yulda's Folly]( https://www.wow-mania.com/armory/?quest=12968)
- [ ] N [79 Daily] [Maintaining Discipline](https://www.wow-mania.com/armory/?quest=13422)  (The Drakkensryd required)

Dun Niffelem
Neutral

- [ ] N [80 Daily] [Hot and Cold ](https://www.wow-mania.com/armory/?quest=12981) (requires Battling the Elements)
- [ ] N [80 Daily] [Blowing Hodir's Horn]( https://www.wow-mania.com/armory/?quest=12977)

Friendly

- [ ] N [80] [Jormuttar is Soo Fat...](https://www.wow-mania.com/armory/?quest=13011)
- [ ] N [80] [Forging a Head]( https://www.wow-mania.com/armory/?quest=12985)
- [ ] N [80] [Mounting Hodir's Helm ](https://www.wow-mania.com/armory/?quest=12987)
- [ ] N [80 Daily] [Polishing the Helm ](https://www.wow-mania.com/armory/?quest=13006)
- [ ] N [80][ Everfrost]( https://www.wow-mania.com/armory/?quest=13420)
- [ ] N [80][ Remember Everfrost!](https://www.wow-mania.com/armory/?quest=13421) (repeatable) 
- [ ] N [80] [Hodir's Tribute](https://www.wow-mania.com/armory/?quest=13559) (repeatable) 

Honored

- [ ] N [80 Daily][ Spy Hunter ](https://www.wow-mania.com/armory/?quest=12994)
- [ ] N [80] [Raising Hodir's Spear ](https://www.wow-mania.com/armory/?quest=13001)
- [ ] N [80 Daily] [Thrusting Hodir's Spear]( https://www.wow-mania.com/armory/?quest=13003)

Revered

- [ ] N [80 Daily] [Feeding Arngrim ](https://www.wow-mania.com/armory/?quest=13046)

Frosthold
Rork Sharpchin has:

- [ ] A [78] [Ancient Relics ](https://www.wow-mania.com/armory/?quest=12870)(repeatable) 

Lagnus has:

- [ ] A [78] [Offering Thanks ](https://www.wow-mania.com/armory/?quest=12863)
- [ ] A [78][ Missing Scouts ](https://www.wow-mania.com/armory/?quest=12864)
- [ ] A [78] [Stemming the Aggressors ](https://www.wow-mania.com/armory/?quest=12866)
- [ ] A [78G3] [Sirana Iceshriek ](https://www.wow-mania.com/armory/?quest=12868)

Fjorlin Frostbrow has:

- [ ] A [78] [Loyal Companions ](https://www.wow-mania.com/armory/?quest=12865)
- [ ] A [78] [Baby Stealers ](https://www.wow-mania.com/armory/?quest=12867)

Archaeologist Andorin has:

- [ ] A [80] [On Brann's Trail ](https://www.wow-mania.com/armory/?quest=12854)
- [ ] A [80][ Sniffing Out the Perpetrator]( https://www.wow-mania.com/armory/?quest=12855)
- [ ] A [80] [Pieces to the Puzzle ](https://www.wow-mania.com/armory/?quest=12858)
- [ ] A [80] [Data Mining ](https://www.wow-mania.com/armory/?quest=12860)
- [ ] A [80] [The Library Console]( https://www.wow-mania.com/armory/?quest=13415)
- [ ] A [80] [Norgannon's Shell ](https://www.wow-mania.com/armory/?quest=12872)
- [ ] A [80] [Aid from the Explorers' League ](https://www.wow-mania.com/armory/?quest=12871)
- [ ] A [80][ The Frostborn King ](https://www.wow-mania.com/armory/?quest=12873)
- [ ] A [80] [Fervor of the Frostborn ](https://www.wow-mania.com/armory/?quest=12874)
- [ ] A [80][ An Experienced Guide]( https://www.wow-mania.com/armory/?quest=12875)

Side quest: - [ ] A [80] [Unwelcome Guests ](https://www.wow-mania.com/armory/?quest=12876)

- [ ] A [80] [The Lonesome Watcher ](https://www.wow-mania.com/armory/?quest=12877)
- [ ] A [80][ Fate of the Titans ](https://www.wow-mania.com/armory/?quest=12986)
- [ ] A [80] [The Hidden Relic]( https://www.wow-mania.com/armory/?quest=12878)
- [ ] A [80] [Fury of the Frostborn King]( https://www.wow-mania.com/armory/?quest=12879)
- [ ] A [80] [The Master Explorer ](https://www.wow-mania.com/armory/?quest=12880)
- [ ] A [80] [The Brothers Bronzebeard]( https://www.wow-mania.com/armory/?quest=12973)
- [ ] A [78 Daily][ Pushed Too Far ](https://www.wow-mania.com/armory/?quest=12869)

Grom'arsh Crash-site

H [78][ Ancient Relics]( https://www.wow-mania.com/armory/?quest=12870) (repeatable)
H [80] [Emergency Measures]( https://www.wow-mania.com/armory/?quest=13000)
H [80] [The Missing Tracker]( https://www.wow-mania.com/armory/?quest=13054)
H [80] [Cave Medicine ](https://www.wow-mania.com/armory/?quest=13055)
H [80] [There's Always Time for Revenge ](https://www.wow-mania.com/armory/?quest=13056)

Boktar Bloodfury has:

H [80] [The Missing Bronzebeard ](https://www.wow-mania.com/armory/?quest=12895)
H [80][ The Nose Knows ](https://www.wow-mania.com/armory/?quest=12909)
H [80] [Sniffing Out the Perpetrator]( https://www.wow-mania.com/armory/?quest=12910)
H [80] [Speak Orcish, Man!]( https://www.wow-mania.com/armory/?quest=12913)
H [80] [Speaking with the Wind's Voice]( https://www.wow-mania.com/armory/?quest=12917)
H [80][ Catching up with Brann]( https://www.wow-mania.com/armory/?quest=12920)
H [80] [Pieces of the Puzzle]( https://www.wow-mania.com/armory/?quest=12926)
H [80][ Data Mining]( https://www.wow-mania.com/armory/?quest=12927)
H [80] [The Library Console]( https://www.wow-mania.com/armory/?quest=13416)
H [80] [Norgannon's Shell ](https://www.wow-mania.com/armory/?quest=12928)
H [80] [Going After the Core ](https://www.wow-mania.com/armory/?quest=13273)
H [80] [The Core's Keeper]( https://www.wow-mania.com/armory/?quest=13274)
H [80] [Forging the Keystone]( https://www.wow-mania.com/armory/?quest=13285)

Camp Tunka'lo

H [80] [Xarantaur, the Witness ](https://www.wow-mania.com/armory/?quest=13426)
H [80] [The Witness and the Hero ](https://www.wow-mania.com/armory/?quest=13034)
H [80] Memories of Stormhoof]( https://www.wow-mania.com/armory/?quest=13037)
H [80] [Distortions in Time](https://www.wow-mania.com/armory/?quest=13038) 
H [80] [Where Time Went Wrong](https://www.wow-mania.com/armory/?quest=13048)
H [80] [The Hero's Arms ](https://www.wow-mania.com/armory/?quest=13049)
H [80] [Changing the Wind's Course ](https://www.wow-mania.com/armory/?quest=13058)

Bouldercrag Refuge
After completing Norgannon's Shell:

H [80] [The Earthen of Ulduar]( https://www.wow-mania.com/armory/?quest=12929) or A [80] [The Exiles of Ulduar ](https://www.wow-mania.com/armory/?quest=12885)
- [ ] N [80] [Rare Earth ](https://www.wow-mania.com/armory/?quest=12930)
- [ ] N [80] Relief for the Fallen https://www.wow-mania.com/armory/?quest=12937 closes N [80][ Fighting Back ](https://www.wow-mania.com/armory/?quest=12931)
- [ ] N [80] [Slaves of the Stormforged](https://www.wow-mania.com/armory/?quest=12957 ) closes N [80] [The Dark Ore ](https://www.wow-mania.com/armory/?quest=12964)
- [ ] N [80] [The Gifts of Loken](https://www.wow-mania.com/armory/?quest=12965)  closes N [80] [Facing the Storm ](Facing the Storm https://www.wow-mania.com/armory/?quest=12978)

While killing golems, loot the [Dark Armor Plate](https://www.wow-mania.com/armory/?item=42203) to continue the quest chain. 

- [ ] N [80][ Armor of Darkness ](https://www.wow-mania.com/armory/?quest=12979)
- [ ] N [80][ The Armor's Secrets]( https://www.wow-mania.com/armory/?quest=12980)
- [ ] N [80] [Valduran the Stormborn ](https://www.wow-mania.com/armory/?quest=12984)
- [ ] N [80][ Destroy the Forges! ](https://www.wow-mania.com/armory/?quest=12988) closes N [80] [Hit Them Where it Hurts ](https://www.wow-mania.com/armory/?quest=12991)
- [ ] N [80][ A Colossal Threat ](https://www.wow-mania.com/armory/?quest=12993)
- [ ] N [80] [The Heart of the Storm ](https://www.wow-mania.com/armory/?quest=12998)
- [ ] N [80] [The Iron Colossus ](https://www.wow-mania.com/armory/?quest=13007)

**Comment by WhitephoenixTheBunny** [comment]
Update of quest item droping
![image](https://user-images.githubusercontent.com/31921098/157453936-5c8f7f9c-6661-4a0c-9765-e02b43d71edf.png)

**Comment by WhitephoenixTheBunny** [comment]
Update of item dropping
![image](https://user-images.githubusercontent.com/31921098/157454092-9f90d0a9-2313-4e7a-9486-1d2f31891722.png)

**Comment by WhitephoenixTheBunny** [comment]
For an update, doing questline on Live and PTR, both are standing on same point.

I am doing [Forging an Alliance](https://www.wow-mania.com/armory/?quest=12924) atm and this is what is happening while talking with [King Jokkum](https://www.wow-mania.com/armory/?npc=30105): 
![image](https://user-images.githubusercontent.com/31921098/157641996-840242c3-8a03-4393-beb1-f0ebe70f829c.png)

[Forging an Alliance](https://www.wow-mania.com/armory/?quest=12924), requires to talk with [King Jokkum](https://www.wow-mania.com/armory/?npc=30105) who should provide **ONLY** [You Can't Miss Him](https://www.wow-mania.com/armory/?quest=12966), this quest need be completed, as the follow up [Battling the Elements](https://www.wow-mania.com/armory/?quest=12967) to complete [Forging an Alliance](https://www.wow-mania.com/armory/?quest=12924)  also.

![image](https://user-images.githubusercontent.com/31921098/157642916-6bd00967-6111-4515-b1b7-53fe3048570a.png)
- "Talk with King Jokkum" is/should be complete once [You Can't Miss Him](https://www.wow-mania.com/armory/?quest=12966) is taken;
- [Fjorn's Anvil Brought to Dun Niffelem](https://www.wow-mania.com/armory/?npc=30126) is/should be complete when [Battling the Elements](https://www.wow-mania.com/armory/?quest=12967) is complete;

---xxx---

[Jormuttar is Soo Fat...]( https://www.wow-mania.com/armory/?quest=13011) and [In Memoriam ](https://www.wow-mania.com/armory/?quest=12975) should **ONLY** be offered by [King Jokkum](https://www.wow-mania.com/armory/?npc=30105) after [Krolmir, Hammer of Storms]( https://www.wow-mania.com/armory/?quest=13010)

![image](https://user-images.githubusercontent.com/31921098/157645386-9e4a25d8-7f03-407c-965e-d7b9ad0884d5.png)
in [wowwiki](https://wowwiki-archive.fandom.com/wiki/Quest:Jormuttar_is_Soo_Fat...)

@mfranca0009 here's where the questline start to mess up

**Comment by WhitephoenixTheBunny** [comment]
This on PTR:

1. I have pick the 3 quest that [King Jokkum](https://www.wow-mania.com/armory/?npc=30105) as to offer:

[You Can't Miss Him](https://www.wow-mania.com/armory/?quest=12966)
[Jormuttar is Soo Fat...]( https://www.wow-mania.com/armory/?quest=13011)
[In Memoriam ](https://www.wow-mania.com/armory/?quest=12975)

2. I have completed:

[You Can't Miss Him](https://www.wow-mania.com/armory/?quest=12966)
[Battling the Elements](https://www.wow-mania.com/armory/?quest=12967) 
[Forging an Alliance](https://www.wow-mania.com/armory/?quest=12924)

[Jormuttar is Soo Fat...]( https://www.wow-mania.com/armory/?quest=13011)
[In Memoriam ](https://www.wow-mania.com/armory/?quest=12975)
Those 2 still on quest log.

3. While turning [Forging an Alliance](https://www.wow-mania.com/armory/?quest=12924) to [Njormeld](https://www.wow-mania.com/armory/?npc=30127), I was able to take from him:

[A New Beginning](https://www.wow-mania.com/armory/?quest=13009)
[Forging a Head](https://www.wow-mania.com/armory/?quest=12985)

[Jormuttar is Soo Fat...]( https://www.wow-mania.com/armory/?quest=13011)
[In Memoriam ](https://www.wow-mania.com/armory/?quest=12975)
Those 2 still on quest log.

4. Completed the 4 quests 

![image](https://user-images.githubusercontent.com/31921098/157649612-f95e80b4-f792-4951-8e2e-d40638dc923e.png)


5. At [King Jokkum](https://www.wow-mania.com/armory/?npc=30105):
Turning [In Memoriam ](https://www.wow-mania.com/armory/?quest=12975) gave me [A Monument to the Fallen](https://www.wow-mania.com/armory/?quest=12976)
 Turned [Jormuttar is Soo Fat...]( https://www.wow-mania.com/armory/?quest=13011)
 At [Njormeld](https://www.wow-mania.com/armory/?npc=30127):
Turning [Forging a Head](https://www.wow-mania.com/armory/?quest=12985) gave me [Mounting Hodir's Helm](https://www.wow-mania.com/armory/?quest=12987)
Turned  [A Monument to the Fallen](https://www.wow-mania.com/armory/?quest=12976) 

6. When going turn [A New Beginning](https://www.wow-mania.com/armory/?quest=13009) at [Thorim](https://www.wow-mania.com/armory/?npc=29445), he will only give me [Veranus](https://www.wow-mania.com/armory/?quest=13050)
From there I did the questline: 
![image](https://user-images.githubusercontent.com/31921098/157652685-4afd10d2-6569-4772-8aaa-ed2c81c6114a.png)

7. [Thorim] [url](https://www.wow-mania.com/armory/?npc=30295) is correcctly offering the quest:

![image](https://user-images.githubusercontent.com/31921098/157652797-d7fbe05f-7d6a-4793-9188-fff7cbbf19a7.png)


I will start a new Testing from this point.


As we can see quests are beeing able to complete without the correct order

**Comment by WhitephoenixTheBunny** [comment]
This is also for @wow-mania as we need edit Armory:

[Loken's Lackeys](https://www.wow-mania.com/armory/?quest=13035) and [The Earthen Oath](https://www.wow-mania.com/armory/?quest=13005) are **both needed** to open [The Reckoning](https://www.wow-mania.com/armory/?quest=13047)

So, on armory at:
- [Loken's Lackeys](https://www.wow-mania.com/armory/?quest=13035) - the "closes quest [The Earthen Oath](https://www.wow-mania.com/armory/?quest=13005)" should be removed
![image](https://user-images.githubusercontent.com/31921098/157655896-7c094f8c-5c06-47f3-a9e9-affe335c172e.png)
- [The Earthen Oath](https://www.wow-mania.com/armory/?quest=13005) - the "closes quest ...." should be removed, and add the "open quest...[The Reckoning](https://www.wow-mania.com/armory/?quest=13047)"
![image](https://user-images.githubusercontent.com/31921098/157656437-434616d9-550f-4db9-89b1-7eaf51c62ec3.png)
- [The Reckoning](https://www.wow-mania.com/armory/?quest=13047) - add the quest The Reckoning](https://www.wow-mania.com/armory/?quest=13047) as requirement
![image](https://user-images.githubusercontent.com/31921098/157656510-48b78b89-69c0-454d-b83a-d4460f44f494.png)

**Comment by WhitephoenixTheBunny** [comment]
Edited name as a Storyline can have several questlines

**Comment by Rushor** [comment]
Forging an Alliance quest will now only be shown if quest: You Can't Miss & Battling the Elements is rewarded

**Comment by Rushor** [comment]
Krolmir, Hammer of Storms will now only be shown if quest: Jormuttar is Soo Fat... & In Memoriam & A Monument to the Fallen is rewarded

**Comment by GMZinius** [comment]
> Forging an Alliance quest will now only be shown if quest: You Can't Miss & Battling the Elements is rewarded

Zirion is testing this atm @Rushor 
PMing you the information

**Comment by Rushor** [comment]
alright should be fixed exactly the other way around

**Comment by Rushor** [comment]
done, exactly the opposite like said in chat :)

**Comment by GMZinius** [comment]
> done, exactly the opposite like said in chat :)

Great news, Will confirm soon, Thank you @Rushor

**Comment by GMZinius** [comment]
@Rushor 

Zirion could pick up [Forging an Alliance](https://wowwiki-archive.fandom.com/wiki/Quest:Forging_an_Alliance) now but couldnt pick [You Can't Miss Him](https://wowwiki-archive.fandom.com/wiki/Quest:You_Can%27t_Miss_Him) and of course not [Battling the Elements](https://wowwiki-archive.fandom.com/wiki/Quest:Battling_the_Elements)

has to be in this order please 

![image](https://user-images.githubusercontent.com/108176634/186028987-5693c9d6-d82f-42d9-ad08-032dd1fafea2.png)

**Comment by Rushor** [comment]
Updated:

Only show quest: You Can't Miss Him    if quest: Forging an Alliance is taken.
+
Only show quest: Battling the Elements if quest: Forging an Alliance is taken.

So the 2 quests will only be shown if player has Forging an Alliance is active and not completed in quest log

**Comment by GMZinius** [comment]
> Updated:
> 
> Only show quest: You Can't Miss Him if quest: Forging an Alliance is taken. + Only show quest: Battling the Elements if quest: Forging an Alliance is taken.
> 
> So the 2 quests will only be shown if player has Forging an Alliance is active and not completed in quest log

Brilliant, All checked and correct working as it should now, Thank you @Rushor

**Comment by Greywolf59** [comment]
I am sorry but my head goes around from this thread so I pardon me for skipping the reading and jumping right to the point what is wrong at the moment.
This will tackle Sons of Hodir/Dun Niffelem.

After getting honored and opening the Sons of Hodir rep and acquiring quest [Forging an Alliance](https://www.wow-mania.com/armory/?quest=12924), player si to return to now friendly Dun Niffelem. From the very 1st arrival there are some issues visible:

1. [Fjorn's Anvil](https://www.wowhead.com/wotlk/object=192071/fjorns-anvil#starts) is visible althoguh logically it should not be there since it is objective of the very 1st visit to Dun Niffelem to actually rescue it from Fire Elementals and bring it back. Does not offer daily yet, which is good.

2. [Hodir's Horn](https://www.wow-mania.com/armory/?object=192078) is visible and offers the daily [Blowing Hodir's Horn](https://www.wow-mania.com/armory/?quest=12977). The object should be visible only after completing [A Monument to the Fallen](https://www.wow-mania.com/armory/?quest=12976) (to actually reassemble the horn).
https://www.wowhead.com/wotlk/object=192078/hodirs-horn#comments:id=524550
The quest should also be only available after completing A Monument to the Fallen by logic.
https://www.wowhead.com/wotlk/quest=12977/hodirs-call#comments:id=485547:reply=96689

3. [Hodir's Spear](https://www.wow-mania.com/armory/?object=192079) is visible although not interactable. it should be visible only after completing [Raising Hodir's Spear](https://www.wow-mania.com/armory/?quest=13001).
https://wowpedia.fandom.com/wiki/Hodir's_Spear?so=search (notes; quest renamed)

4. [Hodir's Helm](https://www.wow-mania.com/armory/?object=192080). Same as above. Should be visible only after completing [Mounting Hodir's Helm](https://www.wow-mania.com/armory/?quest=12987)
https://wowpedia.fandom.com/wiki/Hodir's_Helm?so=search (quest renamed)

**Comment by Lumber-Jack-2** [comment]
for the latest issue, you may can open up a new issue @Greywolf59
